app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.kt

```diff
@@ -0,0 +1,238 @@
+package fr.free.nrw.commons.navtab
+
+import android.accounts.AccountManager
+import android.annotation.SuppressLint
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import fr.free.nrw.commons.AboutActivity
+import fr.free.nrw.commons.BuildConfig
+import fr.free.nrw.commons.CommonsApplication
+import fr.free.nrw.commons.CommonsApplication.ActivityLogoutListener
+import fr.free.nrw.commons.R
+import fr.free.nrw.commons.WelcomeActivity
+import fr.free.nrw.commons.actions.PageEditClient
+import fr.free.nrw.commons.databinding.FragmentMoreBottomSheetBinding
+import fr.free.nrw.commons.di.ApplicationlessInjection
+import fr.free.nrw.commons.feedback.FeedbackContentCreator
+import fr.free.nrw.commons.feedback.FeedbackDialog
+import fr.free.nrw.commons.feedback.model.Feedback
+import fr.free.nrw.commons.kvstore.BasicKvStore
+import fr.free.nrw.commons.kvstore.JsonKvStore
+import fr.free.nrw.commons.logging.CommonsLogSender
+import fr.free.nrw.commons.profile.ProfileActivity
+import fr.free.nrw.commons.review.ReviewActivity
+import fr.free.nrw.commons.settings.SettingsActivity
+import io.reactivex.Single
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+import javax.inject.Inject
+import javax.inject.Named
+
+
+class MoreBottomSheetFragment : BottomSheetDialogFragment() {
+
+    @Inject
+    lateinit var commonsLogSender: CommonsLogSender
+
+    private lateinit var moreProfile: TextView
```

Comment:
can be removed and no need to used lateinit or declare variable when we are on viewbinding we can use kotlin advantage over here